### ISSUE TITLE
parlamint2release: overlapping affiliations

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#704]
I think parlamint2release can be improved with this script:

https://github.com/ufal/ParlaMint-UA/blob/74db5ef0602b8ef9419feca889306cf5772f8de0/Scripts/affiliations-remove-overlaps.xsl

Which remove and merge overlapping affiliations.

Successfully applied in ParlaMint-UA. 
And also in ParlaMint-ES https://github.com/matyaskopp/PARLAMINT-ES-MC/blob/83d523f37f13c7d38f9e3f4d1bdefe4729750fe4/Makefile#L99-L108
```
fix-affiliations: bin/affiliations-remove-overlaps.xsl bin/ParlaMint-UA-lib.xsl
	mv ParlaMint$(DIRSUFFIX)/ParlaMint-ES-listPerson.xml ParlaMint$(DIRSUFFIX)/ParlaMint-ES-listPerson.xml.bak
	$s -xsl:bin/affiliations-remove-overlaps.xsl \
	  ParlaMint$(DIRSUFFIX)/ParlaMint-ES-listPerson.xml.bak \
	  > ParlaMint$(DIRSUFFIX)/ParlaMint-ES-listPerson.xml

bin/affiliations-remove-overlaps.xsl:
	svn export https://github.com/ufal/ParlaMint-UA/trunk/Scripts/affiliations-remove-overlaps.xsl bin/affiliations-remove-overlaps.xsl
bin/ParlaMint-UA-lib.xsl:
	svn export https://github.com/ufal/ParlaMint-UA/trunk/Scripts/ParlaMint-UA-lib.xsl bin/ParlaMint-UA-lib.xsl
```

**Comment by TomazErjavec** [comment]
> I think parlamint2release can be improved with this script

Most likely, but it is somewhat different from the other scripts:

* it includes ParlaMint-UA-lib.xsl and not ParlaMint-lib.xsl
* it doesn't respect the message reporting conventions (INFO:, WARN:., ERROR:)
* it has e.g. mk:pad-date even though et:pad-date already exsits

If it could be somewhat brought in line with the other scripts then, yes, I would be nice if it were included.

**Comment by TomazErjavec** [comment]
I think it's a bit late now to do this change, so moving to future milestone.

**Comment by TomazErjavec** [comment]
This has now been implemented (#851), so, closing.
